packages/fxa-settings/src/components/DropDownAvatarMenu/sign-out.svg

```diff
@@ -0,0 +1 @@
+<svg fill="none" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M6.333 9.26H17M12.733 4.993L17 9.26l-4.267 4.267M9.533 16.727H2.067A1.067 1.067 0 011 15.66V2.86c0-.589.478-1.067 1.067-1.067h7.466" stroke="#111" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
Well, the _stroke_ is set to #111, the fill is set to `none`. If I set the `fill` here to `currentColor`, it looks like this:

![image](https://user-images.githubusercontent.com/13018240/91349520-02c35100-e7ab-11ea-90df-c7a810e9cfed.png)

I'm not sure this one needs a hover state anyway since the text doesn't change color. It could be re-exported later if we want to add a hover color change.

Reply:
Oh right, maybe [`stroke-current`](https://tailwindcss.com/docs/stroke#app) then?

Reply:
Huh, TIL. I've known about `currentColor` on `fill` for years, but I didn't know there was a stroke equivalent 😂 Made the change.